python/rapidsmpf/rapidsmpf/shuffler.pxd

```diff
@@ -30,7 +30,9 @@ cdef extern from "<rapidsmpf/shuffler/shuffler.hpp>" nogil:
         ) except +
         void shutdown() except +
         void insert(unordered_map[uint32_t, cpp_PackedData] chunks) except +
+        void insert_grouped(unordered_map[uint32_t, cpp_PackedData] chunks) except +
```

Comment:
Not sure if I'm missing something from previous discussion, but `insert_grouped` is not in the `pyi`/`pyx` files, is that intentional? Similarly, `insert_chunks` is in `pyi`/`pyx` but not here.

Reply:
Ok, that was in fact my mistake, for some reason I overlooked and thought `insert_grouped` was not even called but it is. Please disregard my comment.